exchanges/kraken/kraken_test.go

```diff
@@ -1265,11 +1255,45 @@ func TestWsAddOrder(t *testing.T) {
 	}
 }
 
-func TestWsCancelOrder(t *testing.T) {
-	setupWsTests(t)
-	if err := k.wsCancelOrders([]string{"1337"}); err != nil {
-		t.Error(err)
+func mockWsCancelOrders(msg []byte, w *websocket.Conn) error {
+	var req WsCancelOrderRequest
+	if err := json.Unmarshal(msg, &req); err != nil {
+		return err
+	}
+	if req.Event != krakenWsCancelOrder {
+		return w.WriteMessage(websocket.TextMessage, []byte(fmt.Sprintf(`{"event":"subscriptionStatus","reqid":%d,"status":"ok"}`, req.RequestID)))
+	}
```

Comment:
This does not get hit, is there something you're trying to do?

Reply:
Yeah, it *was* a protection against when subscriptions were firing.
But then I fixed that with internal/testing/exchange/exchange.go:108-110 and  ...
... dropped this common loot 🤷 

Fixed 596c9cce